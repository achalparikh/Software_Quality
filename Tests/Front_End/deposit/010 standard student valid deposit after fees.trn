04 ACHAL PARIKH        00001 00004.95 S 
