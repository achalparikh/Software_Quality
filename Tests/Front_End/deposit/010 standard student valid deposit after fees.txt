login
standard
ACHAL PARIKH
deposit
00001
5.00
logout
quit