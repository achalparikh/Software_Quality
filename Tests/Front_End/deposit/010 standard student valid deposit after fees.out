Enter user type - "standard" or "admin": 
Logging in as standard user - enter name: 
You are now logged in as "ACHAL PARIKH"
COMMAND: deposit - enter account number:
Depositing to 00001 - enter amount:
ALERT: 0.05 student fee will be deducted.
Deposit of 4.95 - complete!
You are now logged out
